### PR TITLE
Add Prestissimo for PHP services.

### PR DESCRIPTION
**Pull request by dustinleblanc**
@pirog the only thing we might want to doublecheck is that this doesn't
break too many people's existing build steps and decide if we need to
edit teh docz to tell folks this is installed. I think we're safe not to
anyway, but worth a second opinion since I did this so speedy like.

- [x] My code passes relevant CI status checks
- [x] My code includes an update to the `CHANGELOG` ([see here](https://github.com/kalabox/lando/tree/master/docs/changelog))
- [x] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

**Comment by pirog** [comment]
@dustinleblanc how do you envision it breaking peoples build steps?

**Comment by dustinleblanc** [comment]
I don't but I know I just shimmed in an extra step, and maybe if they also are installing it...sadness?

**Comment by dustinleblanc** [comment]
Rebased, ready to go after CI

**Comment by dustinleblanc** [comment]
@pirog this should be ready, AppVeyor failed to package on the PR branch. I *did* a small change to our eslint config to no longer require parens on arrow functions with a single argument, but I can't imagine that is causing a fail here.

```js
(arg) => doThing(arg);
```
became 
```js
arg => doThing(arg);
```

File: plugins/lando-services/services/php/php.js [hunk 1]
```diff
@@ -344,6 +344,9 @@ module.exports = function(lando) {
       }
     });
 
+    // Install Prestissimo
+    config.run_internal.push(['composer', 'global', 'require', 'hirak/prestissimo']);
```
[the remark this answers is not in the document]
Comment (dustinleblanc) [reply plugins/lando-services/services/php/php.js]: done

**Comment by pirog** [comment]
@dustinleblanc fixed this up and merged it with #446 because im lazy right now ;)

**Comment by dustinleblanc** [comment]
Glad to see it worked in :dancer:

**Pull request closed without merging**